File: DataSummitSolution/DataSummitUI/src/app/profileAttributes/models/standardAttribute.ts
import { ProfileAttribute } from "./profileAttribute";

export class StandardAttribute {

	StandardAttributeId: number;
	Name: string;
	ProfileAttributes: ProfileAttribute[];

	constructor(standardAttributeId?: number, name?: string, profileAttributes?: ProfileAttribute[])
	{
		this.StandardAttributeId = standardAttributeId || 0;
		this.Name = name || "";
		this.ProfileAttributes = profileAttributes;
	}
}

File: DataSummitSolution/DataSummitModels/DB/StandardAttribute.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace DataSummitModels.DB
{
    public partial class StandardAttribute
    {
        public StandardAttribute()
        {
            TemplateAttributes = new HashSet<TemplateAttribute>();
        }

        public short StandardAttributeId { get; set; }
        public string Name { get; set; }

        public virtual ICollection<TemplateAttribute> TemplateAttributes { get; set; }
    }
}
